rust/xaynet-server/Cargo.toml

```diff
@@ -66,7 +66,7 @@ tracing = "0.1.19"
 tracing-futures = "0.2.4"
 tracing-subscriber = "0.2.12"
 validator = { version = "0.11.0", features = ["derive"] }
-warp = { git = "https://github.com/xaynetwork/warp" }
+warp = { git = "https://github.com/seanmonstar/warp", branch = "master" }
```

Comment:
Maybe we should pin this to a certain commit. The master branch may contain breaking changes at any point.

Reply:
good idea, i'll change that.